tilt-settings.yaml.example

```diff
@@ -3,3 +3,5 @@ image: <your github handle>/kubewarden-controller
 helm_charts_path: ../helm-charts/
 audit_scanner_path: ../audit-scanner/
 controller_values_file: your-kubewarden-controller-helm-chart-values.yaml
+namespace_labels:
```

Comment:
could you add a comment to describe why we would need to use this setting?

Reply:
done